ush/forecast_postdet.sh

```diff
@@ -556,6 +573,20 @@ FV3_GFS_nml(){
   echo SUB ${FUNCNAME[0]}: FV3 name lists and model configure file created
 }
 
+FV3_GEFS_nml(){
```

Comment:
This is identical to FV3_GFS_nml, why is it needed?

Reply:
There are also a bunch of "GEFS" functions added that just call the GFS version. Is there a need to do that?

Reply:
I added these functions is based on the exglobal_forecast.sh. Such as: Since the function should be similar with GFS functions, I just reuse the GFS functions. 

case $RUN in
  'data') DATM_nml;;
  'gfs') FV3_GFS_nml;;
  'gdas') FV3_GFS_nml;;
  'gefs') FV3_GEFS_nml;;
esac				#no namelist for data atmosphere

Reply:
But `gfs` and `gdas` already both call `FV3_GFS_nml`, why can't `gefs`?

Reply:
Good insight. I can delete FV3_GEFS_nml to use FV3_GFS_nml